### PR TITLE
Add documentation about changing cluster node hostname

### LINKED ISSUE #1162: Process document about changing node hostname
**Component**: Documentation

### PR DESCRIPTION
**Pull request by ycointe**
Issue: GH-1162

**Component**:

**Context**: Add documentation

**Summary**: How to change a node hostname in a cluster

**Acceptance criteria**: 

---

Closes: #1162

### CODE REVIEW

File: docs/changenodehostname.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Change the hostname of one of a cluster node:
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: No `:` in a title

And could be written as `Change the hostname of a cluster node`

File: docs/changenodehostname.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+=============================================
+
+On the node:
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: ditto, no `:`

File: docs/changenodehostname.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+^^^^^^^^^^^^
+
+Change the hostname:
+
+.. code-block:: shell
+
+        $ hostnamectl set-hostname <your hostname, here: ControlPlane2>
+        $ systemctl restart systemd-hostnamed
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst:12]: Do we really need to restart the service?

File: docs/changenodehostname.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        Machine ID: 5003025f93c1a84914ea5ae66519c100
+           Boot ID: f28d5c64f06c48a3a775e24c4f03d00c
+           Virtualization: kvm
+        Oerating System: CentOS Linux 7 (Core)
+           CPE OS Name: cpe:/o:centos:centos:7
+               Kernel: Linux 3.10.0-957.12.2.el7.x86_64
+           Architecture: x86-64 
+
+
+On the bootstrap:
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: `On the bootstrap node` (and no `:`)

File: docs/changenodehostname.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+^^^^^^^^^^^^^^^^^
+
+Verify the hostname change occur a status change on the bootstrap:
+.. code-block:: shell
+        $ kubectl get nodes
+        node2.novalocal    NotReady   etcd,master                   19h       v1.11.7
+
+Change the name of the node in the yaml file used to created it
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: s/yaml/YAML/

File: docs/changenodehostname.rst [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+.. code-block:: yaml
+
+        apiVersion: v1
+        kind: Node
+        metadata:
+          name: controlplane2
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: maybe using `<NEW HOSTNAME>` would make more visible which place needs to be updated.

File: docs/changenodehostname.rst [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+
+
+
+Then apply the configuration: 
+.. code-block:: shell
+
+        $ kubectl apply -f new-node-master3.yaml
+
+Delete the hold node (here node2.novalocal):
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: s/hold/old/, and use quote around the hostname ` ``node2.novalocal`` `

File: docs/changenodehostname.rst [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+                Summary for master.novalocal_master
+                ------------- 
+                Succeeded: 11 (changed=9)
+                Failed:     0
+                -------------
+                Total states run:     11
+                Total run time:  132.435 s
+
+
+On the Control Plane node delete the certificates :
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: No space before `:`

File: docs/changenodehostname.rst [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+        Oerating System: CentOS Linux 7 (Core)
+           CPE OS Name: cpe:/o:centos:centos:7
+               Kernel: Linux 3.10.0-957.12.2.el7.x86_64
+           Architecture: x86-64 
+
+
+On the bootstrap:
+^^^^^^^^^^^^^^^^^
+
+Verify the hostname change occur a status change on the bootstrap:
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: What do you mean here?

File: docs/changenodehostname.rst [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+^^^^^^^^^^^^
+
+Change the hostname:
+
+.. code-block:: shell
+
+        $ hostnamectl set-hostname <your hostname, here: ControlPlane2>
+        $ systemctl restart systemd-hostnamed
+
+Verify if the change is done:
```
Comment (slaperche-scality) [inline docs/changenodehostname.rst]: `Check that the hostname has changed`?

**Review by slaperche-scality — CHANGES_REQUESTED** [review]

File: docs/changenodehostname.rst (hunk 3 above)
Comment (ycointe) [reply docs/changenodehostname.rst:12]: According the documentation I read, yes, it's necessary for the persistence, especially in case of reboot.

**Comment by ycointe** [comment]
Changes and corrections done.

**Review by slaperche-scality — APPROVED** [review]

**Comment by ycointe** [comment]
Decline due to the wrong branch name.

**Pull request closed without merging**